docker/develop/Dockerfile

```diff
@@ -28,6 +28,8 @@ RUN set -e; \
         git ssh tar gzip ca-certificates gnupg \
         # Pythons
         python-pip python3-pip python3-setuptools python-dev \
+        # libpq
+        bison flex \
```

Comment:
These tools should not be required when postgresql is built from a tarball, as it is done in vcpkg. I will suggest a patch.

Reply:
Hey @MBoldyrev! Maybe this 'need fix' suggestion is blocking your 2 PRs from going futher (as Yours other PR is depend on that one, as you mentioned in #3558

Just hanging out here and there looking for some things to work, but noticed that maybe you will find this worth noting.

HF & GL!